src/serverlogging.cpp

```diff
@@ -70,7 +70,7 @@ void CServerLogging::operator<< ( const QString& sNewStr )
     {
         // append new line in logging file
         QTextStream out ( &File );
```

Comment:
The log file wants to flush on each write.

Reply:
I agree, I mis-interpreted `File.flush()` which (probably?) only affects the underlying file, but not the `QTextStream`. I've updated this part to call `out.flush()` (but dropped the `File.flush` as I understand [it is redundant in that case](https://github.com/qt/qtbase/blob/5558eb4edb62ddc920fd5f4171e732fb7cbe53a7/src/corelib/serialization/qtextstream.cpp#L72)).

Reply:
As noted above, I'd understood at some point that both were needed - the text stream was buffer and then the underlying file was separately buffered.  The text stream flush gets to into the file but not necessarily to disk, which is why I think it was like this.

Again, as above, that might have been a Qt4 idiom that no longer applies - but it's worth checking if it's applicable in Qt5, at least, as we won't be moving to Qt6 for a long time, I expect.